src/core.js

```diff
@@ -62,7 +62,16 @@ let doc = null;
  * @template T
  */
 const patchFactory = function(run) {
-  return function(node, fn, data) {
+  /**
+   * TODO(moz): These annotations won't be necessary once we switch to Closure
```

Comment:
NTI currently gives a couple of false warnings and some legitimate ones. I expect some of the false positives to be resolved in the next Closure Compiler release, which should be pretty soon.